pkg/convert/tf.go

```diff
@@ -3416,11 +3419,37 @@ func getPackageBlock(name string, prov *configs.RequiredProvider) (*hclwrite.Blo
 	body := block.Body()
 
 	diags := hcl.Diagnostics{}
-	body.SetAttributeValue("baseProviderName", cty.StringVal(name))
+
+	packageNameParts := strings.Split(prov.Source, "/")
+	packageName := packageNameParts[len(packageNameParts)-1]
+	if isTerraformProvider(packageName) {
+		body.SetAttributeValue("baseProviderName", cty.StringVal("terraform-provider"))
+		body.SetAttributeValue("baseProviderVersion", cty.StringVal("0.3.0"))
+
+		var desiredVersion versions.Version
+		desiredVersion, diags = shim.FindTfPackageVersion(prov)
+
+		innerValue := fmt.Sprintf(`{"remote":{"url":"%s","version":"%s"}}`, 
```

Comment:
I wonder if we should depend on tf bridge for building this parameter value. It's simple now, but I know Ian has commented about potentially doing more pre-processing at parameterisation time which would change this value.
Maybe at least leave a comment here about that?

Reply:
Comment added

Reply:
I don't see the comment for this?